python/flare/ironstrings/ironstrings.py

```diff
@@ -161,15 +161,15 @@ def main():
             # print stackstrings found in this function
             f_ss = list(filter(lambda s: s.fva == fva, userData["stackstrings"]))
             cnt_found_ss += len(f_ss)
-            for ss in sorted(f_ss, key=lambda s: s.written_at):
+            for ss in sorted(f_ss, key=lambda s: s.written_at or 0):
                 print_string(ss.fva, ss.written_at, ss.offset, ss.s)
                 # IMPROVEMENT adjust stack frame member size in IDA view
 
             # print globalstrings found in this function
             f_gs = list(filter(lambda g: g.fva == fva, userData["globalstrings"]))
             cnt_found_ss += len(f_gs)
             for gs in sorted(f_gs, key=lambda g: g.written_at):
-                print_string(gs.fva, gs.written_at, gs.offset, gs.s)
+                print_string(gs.fva, gs.written_at, gs.offset, gs.s or 0)

```

Comment:
I have to inspect/test this in more detail, but from a first glance is this change supposed to be one line above?

Reply:
I'm very sorry!! It's my mistake.
No need to change line 172.
Instead, I should change line 171 to the following.
`for gs in sorted(f_gs, key=lambda g: g.written_at or 0):`

Reply:
I will close this pull request and submit new one to fix the above mistake.